src/server/bqjob/job.go

```diff
@@ -140,6 +140,32 @@ func (job *Job) getResultTable() (*bigquery.Table, error) {
 	return table, nil
 }
 
+func (job *Job) GetResultTableForScript() (*bigquery.Table, error){
+
+	client, err := bigquery.NewClient(job.GetCtx(), os.Getenv("DEKART_BIGQUERY_PROJECT_ID"))
```

Comment:
This line duplicates client creation in `job.Run`, [see code](https://github.com/dekart-xyz/dekart/blob/9539c643e0179be8064b74433d1b801f4c5201a6/src/server/bqjob/job.go#L242); it will be easier to maintain if we create client only in one place and save it in a job struct.

Reply:
Injected the client into the job on Job.Run and used the injected client going forward : 
https://github.com/dekart-xyz/dekart/pull/71/commits/f2d4ae37e2425640119495ba2cddcd185a9dc50c